src/lm/arpa-file-parser.h

```diff
@@ -38,7 +38,7 @@ struct ArpaParseOptions {
   enum OovHandling {
     kRaiseError,     ///< Abort on OOV words
     kAddToSymbols,   ///< Add novel words to the symbol table.
-    kReplaceWithUnk, ///< Replace OOV words with <unk>.
+    kReplaceWithUnk,  ///< Replace OOV words with <unk>.
```

Comment:
I just followed the rule "two spaces between code and comments". I'll modify it as what you suggested.